### ISSUE TITLE
Warn the user if unknown commands were provided

### ISSUE THREAD

**Issue by jamesgeorge007** [preactjs/preact-cli#813]
**Do you want to request a _feature_ or report a _bug_?**
Feature request

**What is the current behaviour?**
`preact <unnkown_cmd>` shows up the custom message as from the arg parser.

```md
	
  ERROR
    No command specified.

  Run `$ preact --help` for more info.
```

**What is the expected behaviour?**
It would be great if the user was warned appropriately.

**If this is a feature request, what is motivation or use case for changing the behaviour?**
The current message doesn't make much sense IMO :thinking:

**Comment by jamesgeorge007** [comment]
@lukeed @reznord Thoughts :thinking:

**Comment by lukeed** [comment]
This should be fixed upstream in `sade` – first, can you show us what you entered?

**Comment by jamesgeorge007** [comment]
@lukeed Any unknown argument passed in with `preact` seem to show up the global help message.
For instance, `preact junkcmd`
I would love to implement it within the `sade` repository if you can guide me further :clap:

**Comment by lukeed** [comment]
Thanks @jamesgeorge007 I've already fixed it. It already should have happened, but there was a bug. You helped me realize that sade needs more tests. So I'm doing that now as well and will then publish a new version. Thanks!

**Comment by lukeed** [comment]
Now available in `sade@1.5.1` – thanks for the heads up!
